uaa/src/main/webapp/WEB-INF/spring/saml-idp.xml

```diff
@@ -29,6 +29,14 @@
     <bean id="samlIdpAuthenticationProvider"
         class="org.cloudfoundry.identity.uaa.provider.saml.idp.IdpSamlAuthenticationProvider" />
 
+    <bean id="idmClient" class="com.vmware.identity.idm.client.CasIdmClient">
```

Comment:
all these xmls are actually fragments stiched together. can we keep single idmClient bean and single uaaUrlRetriever, and make use of them both in all these places needed? thanks.